### DISCUSSION TITLE
OSC scale when in window mode

### DISCUSSION THREAD

**Discussion by advice2020** [mpv-player/mpv#14700] [Q&A] [answered]
Linux Debian based distro / GTK based Desktop Environment
MPV 0.38

I would like to be able to have a little more control over the scale of the OSC in both FULL SCREEN and WINDOW modes.

`Side Note:`
I have been trying to create a very simple custom OSC GUI consisting of only timecodes and seekbar but I am stuck because I cannot find anyone with the knowledge to answer some questions about create a simple OSC GUI.  I am aware that there are some 3rd party seekbar only OSCs available but I want mine to look exactly like the default OSC MPV provides.  Anyways I have made some decent progress but have been stuck for a while now.

Anyways, with this OSC / in general with default OSC if you shrink down your MPV window the seekbar and more importantly the timecode text gets very small which makes it hard to read.
I set my
`scalewindowed=3`
`scalefullscreen=3`
I think 2 is the max available but I just put 3 for each, again do not think there is a difference.

The windowed text / seekbar are still smaller than I would like.
Does anyone know of anything else that can be done to increase the font size / seekbar height when smaller window is used?

I saw some "font_size=" items in the manual, but I think these are just for scripts and not options.

Not sure if it would work out, have not really thought about it,  but was going to put in for a request to when using windowed mode keep the height / font size / etc. the same but just adjust the width of the seekbar, if this makes sense.

Anyways if anyone has any ideas of how this could be achieved in the current version, please let me know.
Thank you to anyone who reads this.

**Comment by guidocella** [comment, marked as answer]
`vidscale=no`

**Comment by advice2020** [comment]
Thank you for your response, I tried this a while back but unfortunately did not notice any difference when using smaller windows.  
Does anyone noticed a difference when using this with smaller windows?

If you or anyone else has any other ideas please let me know.

**Reply by qwertychouskie** [reply]
Apparently it doesn't work when set in `~/.config/mpv/mpv.conf`, you need to set it in `~/.config/mpv/script-opts/osc.conf`.  (I also recommend setting `scalewindowed=2.5` in that same file; adjust that number based on your display and preferences.)  That said, while it can keep things consistent at medium-to-large window sizes, it will still be pretty tiny at very small window sizes.
